src/probnum/quad/solvers/acquisition_functions/_integral_variance_reduction.py

```diff
@@ -0,0 +1,75 @@
+"""Integral variance reduction acquisition function for Bayesian quadrature."""
+
+from __future__ import annotations
+
+from typing import Optional, Tuple
+
+import numpy as np
+
+from probnum.quad.solvers._bq_state import BQState
+from probnum.quad.solvers.belief_updates import BQStandardBeliefUpdate
+
+from ._acquisition_function import AcquisitionFunction
+
+# pylint: disable=too-few-public-methods
+
+
+class IntegralVarianceReduction(AcquisitionFunction):
+    r"""The normalized reduction of the integral variance.
+
+    The acquisition function is
+
+    .. math::
+        a(x) &= \mathfrak{v}^{-1}(\mathfrak{v} - \mathfrak{v}(x))\\
+             &= \frac{(\int k(x', x)p(x')\mathrm{d}x')^2}{\mathfrak{v} v(x)}\\
```

Comment:
I feel that `k(x', x)` here is not good notation for the posterior kernel as this is very easily confused with the prior kernel if one only reads the equation.

Reply:
Sure, we can change it. Any suggestion? Perhaps $k_{post}$, $k_\mathcal{D}$ or $k_N$?

Reply:
All of those appear good to me. I've also seen \bar{k} somewhere.